config/configparser/parser.go

```diff
@@ -132,6 +132,14 @@ func (l *ConfigMap) MergeStringMap(cfg map[string]interface{}) error {
 	return l.k.Merge(toMerge)
 }
 
+// MergeAt merges the ConfigMap instance into the "in" instance as a sub map,
+// at the given key path.
+// If all or part of the key path is missing, it will be created.
+// If the key path is `""`, this is equivalent to Merge.
+func (l *ConfigMap) MergeAt(in *ConfigMap, path string) error {
```

Comment:
Fix comment, the merge is into the current instance from "in"

Reply:
I tried that... But does not work... Need the result in my pointer not in the Retrieved pointer. Actually had this before